ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2

```diff
@@ -196,6 +196,10 @@ host_monitoring_cron:
 - name: run docker dns test
   minute: "*/10"
   job: source $HOME/.bashrc; ops-runner -f -s 60 -n csddr.docker.dns.resolution /usr/bin/cron-send-docker-dns-resolution
+# This is checking whether docker have grpc issue
+- name: run docker grpc check
+  minute: "*/10"
+  job: source $HOME/.bashrc; ops-runner -f -s 60 -n csddr.docker.container.grpc /usr/bin/cron-send-docker-grpc
```

Comment:
csddr.docker.container.grpc usually should csdg.docker.container.grpc 

CSDG = Cron-Send-Docker-Grpc